src/header-generator.js

```diff
@@ -0,0 +1,271 @@
+const fs = require('fs');
+const path = require('path');
+const { BayesianNetwork } = require('bayesian-network');
+const { default: ow } = require('ow');
+
+const headerNetworkDefinitionPath = path.join(__dirname, './headerNetworkDefinition.json');
+const inputNetworkDefinitionPath = path.join(__dirname, './inputNetworkDefinition.json');
+const browserHelperFilePath = path.join(__dirname, './browserHelperFile.json');
+
+const browserHttpNodeName = '*BROWSER_HTTP';
+const operatingSystemNodeName = '*OPERATING_SYSTEM';
+const deviceNodeName = '*DEVICE';
+const missingValueDatasetToken = '*MISSING_VALUE*';
+
+const http2SecFetchAttributes = {
+    mode: 'sec-fetch-mode',
+    dest: 'sec-fetch-dest',
+    site: 'sec-fetch-site',
+    user: 'sec-fetch-user',
+};
+
+const http1SecFetchAttributes = {
+    mode: 'Sec-Fetch-Mode',
+    dest: 'Sec-Fetch-Dest',
+    site: 'Sec-Fetch-Site',
+    user: 'Sec-Fetch-User',
+};
+
+function getRandomInteger(minimum, maximum) {
+    return minimum + Math.floor(Math.random() * (maximum - minimum + 1));
+}
+
+function shuffleArray(array) {
+    if (array.length > 1) {
+        for (let x = 0; x < 10; x++) {
```

Comment:
Why it swaps always 10 items and not based on the number of items in the array? Just curious here 🧑‍🎓

Reply:
It was originally named differently and was (and still is) used only for locales, where there can only be 10 at most. Didn't remember to change it when I changed the name, thanks for the catch.